packages/discord.js/src/structures/GuildMember.js

```diff
@@ -56,10 +56,12 @@ class GuildMember extends Base {
 
     /**
      * The role ids of the member
+     * @name GuildMember#_roles
      * @type {Snowflake[]}
      * @private
      */
-    this._roles = [];
+    Object.defineProperty(this, '_roles', { value: [] });
```

Comment:
I think you also need to add `writable` there, otherwise assigning to it will throw. 😅